Improve descriptions tag (#2772)
near/docs: docs/data-infrastructure/tutorials/running-near-lake/lake-start-options.md

---
sidebar_label: Start options
id: lake-start-options

> description: "Learn how to create a NEAR Lake Framework indexer with flexible start options: from a specific block, the latest final block, or resuming after an interruption."

Reviewer: ```suggestion
description: "Learn how to create an indexer using the NEAR Lake Framework."
```
